web/packages/api/src/assets_v2.ts

```diff
@@ -105,11 +106,18 @@ export type RegistryOptions = {
     ethchains: (string | AbstractProvider)[]
     relaychain: string | ApiPromise
     bridgeHub: string | ApiPromise
+    kusama?: KusamaOptions
```

Comment:
Better name than KusamaOptions could be PKBridgeOptions or whatever the name of their bridge is because technically this code path should work with anybridge collocated with us on Asset Hub.

Reply:
I kind of agree, I had the same thought but then decided against it because:

- The P<>K bridge is the only bridge that can actually be tested against.
- Then all the other code should probably also refer to PKBridge right? And it refers to Polkadot-Kusama anyway.

But I'm open to changing it if you think its way better. I just didn't think it made sense to made it generic here and concrete everywhere else.